codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```
@@ -218,8 +224,8 @@ public void goAwayReceived(final int lastKnownStream, long errorCode, ByteBuf de
         for (int i = 0; i < listeners.size(); ++i) {
             try {
                 listeners.get(i).onGoAwayReceived(lastKnownStream, errorCode, debugData);
-            } catch (RuntimeException e) {
-                logger.error("Caught RuntimeException from listener onGoAwayReceived.", e);
+            } catch (Throwable cause) {
```

Why this change? Do we really want to silence errors?